code/game/machinery/big_manipulator.dm

```diff
@@ -146,11 +162,77 @@
 		return ITEM_INTERACT_SUCCESS
 	return ITEM_INTERACT_BLOCKING
 
+/obj/machinery/big_manipulator/multitool_act(mob/living/user, obj/item/multitool/tool)
+	if(!panel_open)
+		return ITEM_INTERACT_BLOCKING
+	wires.interact(user)
+	return ITEM_INTERACT_SUCCESS
+
+/obj/machinery/big_manipulator/multitool_act_secondary(mob/living/user, obj/item/tool)
+	return multitool_act(user, tool)
+
+/obj/machinery/big_manipulator/wirecutter_act(mob/living/user, obj/item/tool)
+	if(!panel_open)
+		return ITEM_INTERACT_BLOCKING
+	wires.interact(user)
+	return ITEM_INTERACT_SUCCESS
+
+/obj/machinery/big_manipulator/wirecutter_act_secondary(mob/living/user, obj/item/tool)
+	return wirecutter_act(user, tool)
+
 /obj/machinery/big_manipulator/RefreshParts()
 	. = ..()
 
 	manipulator_lvl()
 
+/obj/machinery/big_manipulator/mouse_drop_dragged(atom/monkey, mob/user, src_location, over_location, params)
```

Comment:
```suggestion
/obj/machinery/big_manipulator/mouse_drop_dragged(atom/drop_point, mob/user, src_location, over_location, params)
```
We are dragging the manipulator onto some other atom we want to drop the monkey onto